.changeset/fifty-cars-compare.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-kubernetes': minor
+'@backstage/plugin-kubernetes-backend': minor
```

Comment:
Not actually breaking for the backend right?

I think it's best to highlight the breaking changes in the frontend too, just a short notice of the removed stuff. Might even be best to break into two changesets.

Reply:
Those remove files should not be used anywhere as they are just no-op objects but I'll add that they have been removed